tests/run.sh

```diff
@@ -174,6 +174,9 @@ if [ -n "$DO_COVERAGE" ] && [ "$DO_COVERAGE" != 0 ]; then
     sed "1 s~^~require('luacov.runner')('$source_dir/.luacov'); \0~" \
         "$RC_FILE" > "$tmp_files/awesomerc.lua"
     RC_FILE=$tmp_files/awesomerc.lua
+
+    # Use xresources theme for more coverage.
+    sed -i 's~default/theme~xresources/theme~' "$RC_FILE"
```

Comment:
It could be an integration test, looping over the themes, `sed`ing the config, and reload awesome?!

Is there anything easy to iterate of the them dirs?
Could we have something a bit more generic than https://github.com/awesomeWM/awesome/blob/140481a3e566fd2b7c99191fd9e0da9daa3d26f4/docs/06-appearance.md.lua#L33 in `gears`?